frontend/src/components/panels/project_filters_panel/filter_map_components/CheckBox.js

```diff
@@ -2,30 +2,30 @@ import React from 'react';
 import CheckBoxRow from '../filter_map_components/CheckBoxRow';
 
 class CheckBox extends React.Component {
-    constructor(props) {
-        super(props);
-        this.toggleView = this.toggleView.bind(this);
```

Comment:
It looks like changes like these in the commit were just for spacing. This is fine, but maybe we'll address this in a separate PR.

Reply:
I changed the commit to remove those spacing changes. I think my prettier affected all the files I looked through. Whoops.